scripts/release-notes.txt

```diff
@@ -1,4 +1,17 @@
 Release Notes of the ESBMC model checker
+
+*** Version 7.1
+* Added an experimental flag to cache SSA assertions with `--cache-asserts` (SSA caching #827).
+* Support for macOS M1 architecture (Fixed macOS M1 build #810).
+* Additional operators for the goto-contractor with `--goto-contractor` (Goto contractor: support for multiple operators. #852).
+* Interval analysis now supports integer and floating-point interval contraction for relational operators with `--interval-analysis` (Interval analysis fixes #861).
+* Added `atexit` operational model (Atexit model implementation #859).
+* Fixed an issue that caused extern variables to not be linked correctly (Changes in the c_link and contextt to handle extern variables #881).
```

Comment:
`Fixed an issue that caused extern variables to not be linked correctly` -> `Fixed an issue that caused extern variables to be linked incorrectly `